package.json

```diff
@@ -35,9 +35,10 @@
   },
   "peerDependencies": {
     "bedrock": "^1.0.0",
+    "bedrock-account": "^1.0.0",
     "bedrock-did-client": "^1.0.0",
     "bedrock-express": "^2.0.0",
-    "bedrock-identity": "^4.5.0",
+    "bedrock-identity": "^5.0.0",
     "bedrock-key": "^4.0.0"
```

Comment:
move to 5.x?

Reply:
Seriously, this is already done. What is going on?

Reply:
Does it not say `^5.0.0` for you?

Reply:
I'm talking about `bedrock-key` what are you talking about?  Also, is this based on https://github.com/digitalbazaar/bedrock-passport/pull/25?

Reply:
@mattcollier, oh, I thought you were talking about `bedrock-identity`. This was going to be going in before #25 ... do you think it should go after?

Reply:
Makes no difference to me if you're dealing with the merge conflicts.

Reply:
@mattcollier, ok, if you approve this one I'll deal with rebasing the other onto this one.